judgels-backends/judgels-grader-app/Dockerfile

```diff
@@ -19,9 +19,16 @@ ENV RUSTUP_HOME /usr
 ENV XDG_CACHE_HOME /tmp
 ENV JAVA_HOME /usr/lib/jvm/java-11-openjdk-amd64
 ENV PATH $JAVA_HOME/bin:$PATH
+ENV NODE_MAJOR 18
```

Comment:
Nit: this env var is not used for Node / actual grading; it's just for this Dockerfile. I'd suggest just inlining the version (`/node_18.x no distro main ...`) like the other compiler versions here